build/DownloadCFamilyPlugin/DownloadCFamilyPlugin.csproj

```diff
@@ -0,0 +1,20 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net46</TargetFramework>
+    
+    <!-- Not a product project so we don't want it to be taken into account for code coverage -->
```

Comment:
In the long run, we may even want to completely exclude it from SQ analysis.

Reply:
Good point point. I've changed the property to `<SonarQubeExclude>true</SonarQubeExclude> so the entire project will be completely ignored.